hstream-sql/src/HStream/SQL/Internal/Validate.hs

```diff
@@ -719,6 +719,11 @@ instance Validate Insert where
     mapM_ isConstExpr exprs
     return insert
 
+  validate insert@(InsertSelect _ hIdent select) = do
+
+    return insert
+
+
```

Comment:
Is there any reason not to validate the new-added clause? And what are the extra blank lines used for...?